DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -73,6 +73,7 @@
 # Real data raw to digi
 process.load("EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi")
 process.siPixelDigis.IncludeErrors = True
+process.siPixelDigis.UsePhase1 = True
```

Comment:
Hi @fioriNTU - would it not work just to change

process = cms.Process("PIXELDQMLIVE")

to 
from Configuration.StandardSequences.Eras import eras
process = cms.Process("PIXELDQMLIVE", eras.Run2_2017")

as to get all the 2017 configuration changes without having to repeat them in your cfgs?